hk-package-clean-except/action.yml

```
@@ -0,0 +1,94 @@
+        echo ''                                                                            >> clean-except.py
+        echo 'import os'                                                                   >> clean-except.py
+        echo ''                                                                            >> clean-except.py
+        echo 'from ghapi.all import GhApi'                                                 >> clean-except.py
+        echo 'from ghapi.core import print_summary'                                        >> clean-except.py
+        echo 'from ghapi.page import paged'                                                >> clean-except.py
+        echo ''                                                                            >> clean-except.py
+        echo 'org_str = "${{ inputs.package-org }}"'                                       >> clean-except.py
+        echo 'pck_str = "${{ inputs.package-name }}"'                                      >> clean-except.py
+        echo 'valid_tags_str = "${{ inputs.tags-kept }}"'                                  >> clean-except.py
+        echo 'valid_tags = [x.strip() for x in valid_tags_str.split(",")]'                 >> clean-except.py
+        echo ''                                                                            >> clean-except.py
+        echo 'api = GhApi(debug=print_summary, token=os.getenv("PACKAGE_DELETION_TOKEN"))' >> clean-except.py
+        echo ''                                                                            >> clean-except.py
+        echo 'paged_packages = paged('                                                     >> clean-except.py
+        echo '    api.packages.get_all_package_versions_for_package_owned_by_org,'         >> clean-except.py
+        echo '    org=org_str,'                                                            >> clean-except.py
+        echo '    package_name=pck_str,'                                                   >> clean-except.py
+        echo '    package_type="container",'                                               >> clean-except.py
+        echo '    state="active",'                                                         >> clean-except.py
+        echo '    per_page=100,'                                                           >> clean-except.py
+        echo ')'                                                                           >> clean-except.py
+        echo ''                                                                            >> clean-except.py
+        echo '# Loop over all pages'                                                       >> clean-except.py
+        echo 'for page in paged_packages:'                                                 >> clean-except.py
+        echo '    for package in page:'                                                    >> clean-except.py
+        echo '        # Check if the given package has no tags'                            >> clean-except.py

```

```suggestion
        echo '        # Check if the given package has valid tags'               >> clean-except.py

```
May change the indentation.